packages/nextjs/app/builders/[0x575109e921C6d6a1Cb7cA60Be0191B10950AfA6C]/page.tsx

```diff
@@ -0,0 +1,106 @@
+import Image from "next/image";
+import { Metadata } from "next";
+
+export const metadata: Metadata = {
+  title: "Smartdev | Developer Profile",
+  description: "Personal profile page for Smartdev, a Solidity developer passionate about DeFi and DAO governance.",
+};
+
+export default function SmartdevProfilePage() {
```

Comment:
You might want to consider typing your page as NextPage since we are using TypeScript, as this ensures better integration and adds clarity for future maintainers by indicating that the component is a Next.js page.